draft/2022-08-03-this-week-in-rust.md

```diff
@@ -124,44 +114,41 @@ Rusty Events between 2022-08-03 - 2022-08-31 🦀
     * [**Tokyo Rust Game Hack 2022 edition: The Bombercrab Challenge**](https://bombercrab-rust-game-hack.peatix.com/view)
 
-### Europe
-
-* 2022-07-27 | Amsterdam, NL | [Rust Developers Amsterdam Group](https://www.meetup.com/rust-amsterdam-group/)
-    * [**Rust Developers Amsterdam Group**](https://www.meetup.com/rust-amsterdam-group/events/287019877/)
-* 2022-07-28 | Copenhagen, DK | [Copenhagen Rust Group](https://cph.rs/)
-    * [**Hack Night #27**](https://www.meetup.com/copenhagen-rust-meetup-group/events/287057762/)
-* 2022-07-28 | Oslo, NO | [Rust Oslo](https://www.meetup.com/Rust-Oslo/)
-    * [**Rust & Relax @ Brygg w/Jon Gjengset**](https://www.meetup.com/rust-oslo/events/287422914/)
-* 2022-08-02 | London, UK | [Rust London User Group](https://www.meetup.com/Rust-London-User-Group/)
-    * [**Rust London Code Dojo: Rust with Front-End Web Assembly**](https://www.meetup.com/rust-london-user-group/events/287271789/)
 
 ### North America
 
-* 2022-07-27 | Austin, TX, US | [Rust ATX](https://www.meetup.com/rust-atx/)
-    * [**Rust Lunch**](https://www.meetup.com/rust-atx/events/287009519/)
 * 2022-08-05 | Portland, OR, US + Virtual | [RustConf](https://rustconf.com/)
     * [**RustConf 2022**](https://rustconf.com/)
 * 2022-08-06 | Portland, OR, US | [Rust Project Teams](https://www.rust-lang.org/governance)
     * [**RustConf 2022 PostConf Unconf**](https://www.eventbrite.com/e/rustconf-postconf-unconf-registration-373057423797) | [**Blog post**](https://blog.rust-lang.org/2022/06/28/rust-unconference.html)
 * 2022-08-10 | Atlanta, GA, US | [Rust Atlanta](https://www.meetup.com/rust-atl/)
     * [**Grab a beer with fellow Rustaceans**](https://www.meetup.com/rust-atl/events/pczdssydclbnb/)
-* 2022-08-11 | Columbus, IL | [Columbus Rust Society](https://www.meetup.com/columbus-rs/)
+* 2022-08-11 | Columbus, OH, US| [Columbus Rust Society](https://www.meetup.com/columbus-rs/)
     * [**Monthly Meeting**](https://www.meetup.com/columbus-rs/events/dpkhgrydclbpb/)
 * 2022-08-16 | San Francisco, CA, US | [San Francisco Rust Study Group](https://www.meetup.com/san-francisco-rust-study-group/)
     * [**Rust Hacking in Person**](https://www.meetup.com/san-francisco-rust-study-group/events/wjkjssydclbvb/)
 * 2022-08-23 | Toronto, ON, CA | [Rust Toronto](https://www.meetup.com/rust-toronto/)
     * [**WebAssembly plugins in Rust**](https://www.meetup.com/rust-toronto/events/287284601/)
+* 2022-08-25 | Ciudad de México, MX | [Rust MX](https://www.meetup.com/rust-mx/)
+    * [**Concurrencia & paralelismo con Rust**](https://www.meetup.com/rust-mx/events/287561814/)
+* 2022-08-25 | Lehi, UT, US | [Utah Rust](https://www.meetup.com/utah-rust/)
+    * [**Hello World Cargo Crates Using Github Actions with jojobyte and Food!**](https://www.meetup.com/utah-rust/events/kvrxqsydclbpb/)
+
+### Europe
+
+* 202-08-30 | Utrecht, NL | [Rust Nederland](https://www.meetup.com/rust-nederland/)
```

Comment:
```suggestion
* 2022-08-30 | Utrecht, NL | [Rust Nederland](https://www.meetup.com/rust-nederland/)
```

Reply:
Thanks for catching this! Will include this with the re-ordering change